.travis.yml

```diff
@@ -52,5 +52,12 @@ script:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then npm test; fi
   - npm run prebuild-node
   - npm run prebuild-electron
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then docker build -t node-keytar/i386 docker/i386 && docker run --rm -v ${PWD}:/project node-keytar/i386 /bin/bash -c "cd /project && npm run prebuild-node-ia32 && npm run prebuild-electron-ia32 && npm run prebuild-electron-arm64"; fi
+  - |
+    if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+      mkdir -p prebuilds && chmod 777 prebuilds
+      docker build -t node-keytar/i386 docker/i386
```

Comment:
Why build it every time, and not just publish it on docker hub?

Reply:
Having done this on other OSS projects, unless there's a need for this specific setup I prefer having a fast feedback loop by using the same Dockerfile from the repository.

If this were a bigger project, with a more active CI, I'd re-evaluate and maybe get this publishing to Docker Hub.